.zuul.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- job:
+    name: e2e-conformance
```

Comment:
It's better to use the repository name as a prefix for jobs inside a repository. E.g. k8s-cluster-api-provider-e2e-conformance.